File: appAdictive/src/main/java/appAdictive/dto/RutinaPersDTO.java
package appAdictive.dto;

import java.util.Date;
import java.util.List;

import appAdictive.util.enums.InfoTipo;
import appAdictive.util.enums.TipoEj;
import lombok.Data;

@Data
public class RutinaPersDTO {

	private String nombre;

	private InfoTipo descripcionDias;
	
	private Date fechaFinal;

	private TipoEj tipo;
	
	private int kcal;
	
	private int numeroDias;
	
	private List<DiaDTO> dias;
	
	public DiaDTO addDia(DiaDTO dias) {
		getDias().add(dias);
		return dias;
	}

}

File: deportivapp/src/app/models/rutina-pers-dto.ts
import { DiaDTO } from './dia-dto';
export class RutinaPersDTO {
    nombre : string;
    fechaFinal : Date;
    tipo : string;
    kcal:number;
    numeroDias:number;
    dias:DiaDTO[];
    descripcionDias:string;

    constructor(nombre: string, fechaFinal : Date, tipo : string, kcal:number, numeroDias:number,
        dias:DiaDTO[], descripcionDias:string) {
        this.descripcionDias = descripcionDias;
        this.nombre = nombre;
        this.tipo = tipo;
        this.kcal = kcal;
        this.numeroDias = numeroDias;
        this.dias = dias;
        this.fechaFinal = fechaFinal;
    }
}